src/utils/nodes/BinaryExpression.ts

```diff
@@ -0,0 +1,31 @@
+import { NodeClass, NodeClassConstructor } from './Node.abstract';
+
+export class BinaryExpressionClass extends NodeClass {
+	constructor(params: NodeClassConstructor) {
+		super(params);
+	}
+
+	serialize = () => {
+		// eslint-disable-next-line @typescript-eslint/no-explicit-any
+		const binaryExpression = this.node as any;
+		const left = binaryExpression.left.value;
+		const right = binaryExpression.right.value;
+		const operator = binaryExpression.operator;
+		switch (operator) {
+			case '+':
+				return left + right;
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion. -->
🔨 issue(blocking):
There is no guarantee that `binaryExpression ` left and right value is a number.
For example, this expression will send NaN to console:
`console.log(2+(8+3));`

![image](https://github.com/user-attachments/assets/7eb5fb3f-89e5-4172-9135-40d7367b979d)